packages/transition-frontend/src/services/map/events/RoutingSectionMapEvents.ts

```diff
@@ -11,7 +11,9 @@ import serviceLocator from 'chaire-lib-common/lib/utils/ServiceLocator';
 
 /* This file encapsulates map events specific for the 'routing' section */
 
-const isRoutingActiveSection = (activeSection: string) => activeSection === 'routing';
+const isRoutingActiveSection = (activeSection: string) => {
```

Comment:
This might be a naive question, but could the section itself set a flag when it get activated to request the routing type event. Instead of having to need a static list here?